src/triage/component/postmodeling/utils/README.md

```diff
@@ -0,0 +1,19 @@
+## Util for Saving Predictions of Selected Model Groups
+
+The script: `predictions_selected_model_groups.py`
+
+config file example: `add_predictions_example_config.py`
+
+This utility accepts a list of model group ids and an experiment hash, and generates & saves test predictions for the relevant models. This util is primarily targetted to be used post-audition for the following use-case:
+1. A large model grid was used in the experiment with `save_predictions=False`
+2. Audition was used to narrow down model_groups of interest & predictions are required for postmodeling
+
+**Usage**
+
+This util is available with the triage CLI and can be run using the following command
```

Comment:
Would also be good to include usage instructions for importing the function and running in a python script separate from the CLI.

Reply:
added